.goreleaser.yaml

```diff
@@ -31,6 +31,49 @@ builds:
       - amd64
       - arm64
 
+  - id: linux-nsc
+    env:
+      - CGO_ENABLED=0
+    main: ./cmd/nsc
+    binary: nsc
+    flags:
+      - -trimpath
+    ldflags:
+      - -s -w -X namespacelabs.dev/foundation/internal/cli/version.Tag=v{{.Version}}
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm64
+
+  - id: macos-nsc
+    env:
+      - CGO_ENABLED=1
```

Comment:
Ideally we don't need CGO_ENABLE for nsc as it's only needed for fsevents, which we shouldn't be pulling in.